shipment_advice/models/stock_package_level.py

```diff
@@ -23,12 +25,15 @@ def button_load_in_shipment(self):
 
     def _load_in_shipment(self, shipment_advice):
         """Load the package levels into the given shipment advice."""
-        self.is_done = True
```

Comment:
This line should not be removed :thinking: Package level has to be flagged `is_done=True` when it's loaded in the shipment.